README.md

```diff
@@ -93,37 +155,6 @@ Where fixes are available within a rule, these can be applied automatically with
 golangci-kube-api-linter run path/to/api/types --fix
 ```
 
-### Golangci-lint Plugin
-
-The Kube API Linter can also be used as a plugin for `golangci-lint`.
-To do this, you will need to install the `golangci-lint` binary and then install the Kube API Linter plugin.
-
-More information about golangci-lint plugins can be found in the [golangci-lint plugin documentation][golangci-lint-plugin-docs].
-
-[golangci-lint-plugin-docs]: https://golangci-lint.run/plugins/go-plugins/
-
-```shell
-go build -buildmode=plugin -o bin/kube-api-linter.so sigs.k8s.io/kube-api-linter/pkg/plugin
-```
-
-This will create a `kube-api-linter.so` file in the `bin` directory.
-
-The `golangci-lint` configuration is similar to the module configuration, however, you will need to specify the plugin path instead.
-
-```yaml
-linters-settings:
-  custom:
-    kubeapilinter:
-      path: "bin/kube-api-linter.so"
-      description: Kube API LInter lints Kube like APIs based on API conventions and best practices.
-      original-url: sigs.k8s.io/kube-api-linter
-      settings:
-        linters: {}
-        lintersConfig: {}
```

Comment:
Looking at the diff above, it seems we just need to update this section slightly, and the rest remains true and the same?